draft-ietf-privacypass-architecture.md

```diff
@@ -265,46 +252,41 @@ The mechanism by which clients obtain the Issuer public key is not specified.
 Clients may be configured with this key or they may discover it via some other
 form. See {{?CONSISTENCY=I-D.wood-key-consistency}}.
 
-Depending on the use case, issuance may requrie some form of Client
+Depending on the use case, issuance may require some form of Client
 anonymization service similar to an IP-hiding proxy so that Issuers cannot
 learn information about Clients. This can be provided by an explicit
 participant in the issuance protocol, or it can be provided via external means,
 e.g., through the use of an IP-hiding proxy service like Tor. In general,
 Clients SHOULD minimize or remove identifying information where possible when
 invoking the issuance protocol.
 
-Issuers MUST NOT issue tokens for Clients through untrusted Mediators. This is
-important because the Mediator's role is to vouch for trust in
+Issuers MUST NOT issue tokens for Clients through untrusted Attesters. This is
+important because the Attester's role is to vouch for trust in
 privacy-sensitive Client information, such as account identifiers or IP address
 information, to the Issuer. Tokens produced by an Issuer that admits issuance
-for any type of mediation cannot be relied on for any specific property.
-See {{attestation}} for more details.
-
-### Mediator Role
-
-Mediation is an important part of the issuance protocol. Mediaton involves
-two high level functions: attestation and accounting. These are described in
-more detail below.
+for any type of attestation cannot be relied on for any specific property.
+See {{attester-role}} for more details.
 
-#### Attestation
+### Attester Role
 
-Attestation is the process by which the Client's bear witness, confirm, or
-authenticate so as to demonstrate a certain property about themselves that are
-used during issuance. Examples of attestation properties include, though are
-not limited to:
+Attestation is an important part of the issuance protocol. Attestation is the
+process by which the Clients bear witness, confirm, or authenticate so as to
```

Comment:
`bear witness` seems strange. What can a Client bearing witness to about itself? Maybe this should be `by which an Attester bears witness to, confirms, or authenticates a Client so as to verify a property about the Client that is required for Issuance.`

Reply:
Yeah, that's better!